static/app/components/core/alert.stories.tsx

```diff
@@ -27,89 +37,33 @@ export default storyBook('Alert', (story, APIReference) => {
         <p>
           The default <JSXNode name="Alert" /> looks like this:
         </p>
-        <Alert type="info">Sentry is cool!!</Alert>
-        <p>
-          You can even add in links, which will be nicely formatted for you inside the
-          alert:
-        </p>
-        <Alert type="info">
-          Sentry is cool!{' '}
-          <ExternalLink href="https://sentry.io/welcome/">Learn more here.</ExternalLink>
-        </Alert>
-      </Fragment>
-    );
-  });
-
-  story('type', () => {
-    return (
-      <Fragment>
-        <p>
-          The <JSXProperty name="type" value /> prop specifies the alert category, and
-          changes the color and default icon of the alert.
-        </p>
         <Alert.Container>
-          <Alert type="error" showIcon>
-            This is an error alert. Something went wrong.
-          </Alert>
-          <Alert type="info" showIcon>
-            Info alert. Put some exciting info here.
-          </Alert>
-          <Alert type="muted" showIcon>
-            Muted alerts look like this.
-          </Alert>
-          <Alert type="success" showIcon>
-            Success alert. Yay!
-          </Alert>
-          <Alert type="warning" showIcon>
-            Warning alert. Something is about to go wrong, probably.
-          </Alert>
```

Comment:
Some of these felt excessive, links are just children (+ we have an AlertLink component)

Reply:
Nice cleanup, thanks!